free-1.md

```diff
@@ -58,11 +80,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Their time would be spent more on the systems aspects of the project, maybe with a 75% to 25% compared to the language aspects, because the project has to primarily focus on working fully functional and in the best way to help the user, rather than the design of the language.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I believe it is a big idea, because I still know people who use "bookies" to do sports betting because they do not fully understand all the concepts and terms. It is very ambitious as well because it would take an extreme amount of work to perfectly capture the user's requests and turn them into actual bets.
```

Comment:
How might this language make betting easier for bookies? Just a question I though I might ask.